### ISSUE TITLE
Adopt accessible view in terminal

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#192338]
This change should be seamless for screen reader users and we'll want to test it thoroughly to ensure that.

**Comment by meganrogge** [comment]
To do: consider adding actions to the toolbar like open detected link, go to cwd, run recent command, and more for discoverability purposes.
